### ISSUE TITLE
Ques about height appending

### ISSUE THREAD

**Issue by huixiancheng** [guochengqian/PointNeXt#6]
Hi, Thank you for such great open source work!
Since I' m new to this area, I' m confused about this part of the paper mentioned：**height appending [47] (i.e., appending the measurement of each point along the gravity direction of objects as additional input features)**. 
If no wrong, it seems to correspond to these two parts of code in openpoints.

[s3dis](https://github.com/guochengqian/openpoints/blob/fb998885f2895922ce257affa4eda81ff12a615b/dataset/s3dis/s3dis.py#L136-L139)
```
        # pre-process.
        if self.transform is not None:
            data = self.transform(data)
        data['x'] = torch.cat((data['x'], torch.from_numpy(
            coord[:, 3-self.n_shifted:3].astype(np.float32))), dim=-1)
```
[scanobjectnn](https://github.com/guochengqian/openpoints/blob/fb998885f2895922ce257affa4eda81ff12a615b/dataset/scanobjectnn/scanobjectnn.py#L90-L95)
```
        # height appending. @KPConv
        if 'heights' in data.keys():
            data['x'] = torch.cat((data['pos'], data['heights']), dim=1)
        else:
            data['x'] = torch.cat((data['pos'],
                                   torch.from_numpy(current_points[:, self.gravity_dim:self.gravity_dim+1] - current_points[:, self.gravity_dim:self.gravity_dim+1].min())), dim=1)
```
If I understand correctly, gravity_dim refers to the z-axis of the point cloud. The value on the height could be the original z value (for s3dis), or its difference from the height minimum (for scanobjectnn). Is my understanding correct?

**Comment by guochengqian** [comment]
hi both of them are the same: 1) along gravity dimension; 2) relative value. Note in s3dis, the minimum z is 0.

**Comment by huixiancheng** [comment]
Thx for your kindly reply.
